### ISSUE TITLE
time for a new 5.2-0 release?

### ISSUE THREAD

**Issue by tom077** [phpvirtualbox/phpvirtualbox#91]
5.2-0-rc1 is 3 months old now. There are a lot of improvements in the develop branch.

**Comment by h6w** [comment]
I agree.  Sorry.  I have been busy working on other projects with deadlines.  I will release one shortly.

**Comment by Neustradamus** [comment]
Can you confirm if the latest 5.2.x (EOL): https://www.virtualbox.org/wiki/Changelog-5.2: VirtualBox 5.2.44 (released July 14 2020)?
- https://github.com/phpvirtualbox/phpvirtualbox/tree/master
